data/working_groups.yml

```diff
@@ -0,0 +1,74 @@
+---
+# template
+# - title:
+#   goal:
+#   open_letter:
+#   communication_channel:
+#   url:
+#   chairs:
+#     - name:
+#       email:
+#     - name:
+#       email:
+#   status:
+#   outcome:
+#       - url:
+#       - url:
+#
+
+# ACTIVE
+
+-   title: BIDS 2.0
+    goal: Discussions and suggestions of backwards incompatible changes to BIDS
+    open_letter: TBD
+    communication_channel: TBD
+    url: https://github.com/bids-standard/bids-2-devel
+    chairs:
+    -   name: Yaroslav Halchenko
+        email: debian@onerussian.com
+    status: active
+
+-   title: Governance amendments
+    goal: Clarification of the BEP process
+    open_letter: TBD
+    communication_channel: TBD
+    url:
+    chairs:
+    -   name: Ariel Rokem
+        email: arokem@uw.edu
+    status: active
+
+-   title: Code of conducts amendments
+    goal: Update and uniformizations of the BIDS code of conducts
+    open_letter: TBD
+    communication_channel: TBD
+    url:
+    chairs:
+    -   name: Rémi Gau
+        email: remi.gau@gmail.com
+    status: active
+
+-   title: BIDS online presence
+    goal: Streamline online resources based on community feedback
+    open_letter: TBD
+    communication_channel: TBD
+    url:
```

Comment:
@christinerogers @ericearl Can you give me some links for the open letter, a communication channel where people can join and interact with the working group and URL that points to where the work is happening

Reply:
Still not sure what is the open document or URL that I should put in here: will leave as TBD